.gitignore

```diff
@@ -5,6 +5,9 @@ bin-int/
 
 # Hazel files
 *.log
+HazelProfile-Startup.json
+HazelProfile-Runtime.json
+HazelProfile-Shutdown.json
```

Comment:
Do not add these to `.gitignore`. Cherno does not want them in the code, later he only wants a single folder with all the rubbish here. Hence why the code on github is pushed without these files and with the profiler off, such that PRs do not carry profile results.

Reply:
But the problem is: everytime i want to push new code and i ran it with profiling turned on i have to delete those files manually to prevent them from being pushed. and that sucks.